.github/workflows/CI.yml

```diff
@@ -39,8 +39,6 @@ jobs:
       contents: read
     runs-on: ${{ matrix.os }}
     timeout-minutes: 90
-    env:
-      JULIA_CONDAPKG_BACKEND: "Null"
```

Comment:
This was the hack around the original encounter with this error that this PR fixes, so it can be removed, as well as used as confirmation that the fix in this PR should resolve the issue generally.